### PR TITLE
Make travis work

### PR DESCRIPTION
**Pull request by SwampertX**
because I prefer green build badges :smiley: 

## Description

Trying to make travis work - except for tests. Will re-enable tests after UT are fixed. #15 

## Motivation and Context

Fix CI for cinnabot

## How Has This Been Tested?

Travis will check

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by SwampertX** [comment]
Ready for review.

Currently the build does not error (due to wrong configuration), but fails (due to processes exiting with non-zero exit codes), which is a great sign (and all this PR is trying to do).

**Comment by hongshaoyang** [comment]
Closes #26

File: spaces_test.go [hunk 1]
```diff
@@ -3,25 +3,11 @@ package cinnabot
 import (
 	"testing"
 
-	"github.com/stretchr/testify/mock"
 	tgbotapi "gopkg.in/telegram-bot-api.v4"
 )
 
 //to run this test: go test spaces_test.go cinnabot.go spaces.go
 
-type mockBot struct {
```
Comment (SwampertX) [inline spaces_test.go]: Just curious: is there a reason why these lines are removed instead of commented out?

Comment (chloelee767) [reply spaces_test.go]: I think its because its a duplicate - the mockBot struct is defined in basic_test.go too.

Comment (SwampertX) [reply spaces_test.go]: Ahh I see!

**Pull request merged**